.gitignore

```diff
@@ -1,2 +1,31 @@
 npm-debug.log
 node_modules/
+
+#### OSX
```

Comment:
This list of things is really better suited for your own personal global gitignore file, instead of being in this repository.